### PR TITLE
Ts support; Fix account, customFee types

### PR DESCRIPTION
**Pull request by arufaresearch**
+ Junokit types.
+ Ts script for sample project.
+ Fix account and customFee types.
+ Add customFee for juno signingClient as not directly support by cosmjs lib.

### CODE REVIEW

File: packages/junokit/package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "junokit",
-  "version": "0.0.5",
-  "main": "dist/index.js",
+  "version": "0.0.6",
+  "main": "dist/src/index.js",
```
Comment (amityadav0) [inline packages/junokit/package.json:4]: Please restore this. I optimized the build.

File: packages/junokit/package.json [hunk 2]
```diff
@@ -41,12 +41,7 @@
   "engines": {
     "node": ">=14.11.0"
   },
-  "files": [
-    "dist/",
-    "LICENSE",
-    "README.md",
-    "sample-project/"
-  ],
+  "files": ["build/", "README.md", "sample-project/"],
```
Comment (amityadav0) [inline packages/junokit/package.json:44]:
```suggestion
  "files": ["dist/", "README.md", "sample-project/"],
```

File: packages/junokit/tsconfig.json [hunk 3]
```diff
@@ -69,7 +69,5 @@
     "skipLibCheck": true,                           /* Skip type checking of declaration files. */
     "forceConsistentCasingInFileNames": true        /* Disallow inconsistently-cased references to the same file. */
   },
-  "include": [
```
Comment (amityadav0) [inline packages/junokit/tsconfig.json:72]: Please restore this as well

File: packages/junokit/tsconfig.json [hunk 4]
```diff
@@ -69,7 +69,5 @@
...
-    "src"
-  ]
+  "exclude": ["sample-project"]
```
Comment (amityadav0) [inline packages/junokit/tsconfig.json:72]: why are we excluding sample-project ?
I think we need it in build

File: packages/junokit/package.json (hunk 1 above)
Comment (arufaresearch) [reply packages/junokit/package.json:4]: Restore how?

Comment (amityadav0) [reply packages/junokit/package.json:4]: `"main": "dist/src/index.js",` -> `"main": "dist/index.js",`

**Pull request merged**